packages/clay-list-group/src/ClayListGroup.js

```diff
@@ -0,0 +1,132 @@
+import 'clay-dropdown';
+import 'clay-icon';
+import 'clay-label';
+import 'clay-link';
+import 'clay-sticker';
+
+import Component from 'metal-component';
+import defineWebComponent from 'metal-web-component';
+import Soy from 'metal-soy';
+import {Config} from 'metal-state';
+
+import templates from './ClayListGroup.soy.js';
+
+/**
+ * Metal ClayListGroup component.
+ */
+class ClayListGroup extends Component {
+	/**
+	 * Continues the propagation of the checkbox changed event
+	 * @param {!Event} event
+	 * @private
+	 */
+	handleItemCheckboxClick_(event) {
+		this.emit('itemToggled', event);
+	}
+}
+
+/**
+ * State definition.
+ * @static
+ * @type {!Object}
+ */
+ClayListGroup.STATE = {
+	/**
+	 * CSS classes to be applied to the element.
+	 * @instance
+	 * @memberof ClayListGroup
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	elementClasses: Config.string(),
+
+	/**
+	 * Id to be applied to the element.
+	 * @instance
+	 * @memberof ClayListGroup
+	 * @type {?string|undefined}
+	 * @default undefined
+	 */
+	id: Config.string(),
+
+	/**
+	 * List of items.
+	 * @instance
+	 * @memberof ClayListGroup
+	 * @type {!array}
+	 * @default undefined
+	 */
+	items: Config.arrayOf(
+		Config.shapeOf({
+			actionItems: Config.arrayOf(
+				Config.shapeOf({
+					disabled: Config.bool().value(false),
+					events: Config.object(),
+					indicatorSymbol: Config.string(),
+					label: Config.string().required(),
+					quickAction: Config.bool(),
+					separator: Config.bool().value(false),
+					url: Config.string().required(),
+				})
+			),
+			description: Config.string(),
+			icon: Config.string(),
+			iconShape: Config.oneOf(['circle', 'rounded']).value('rounded'),
+			label: Config.string(),
+			labelStyle: Config.oneOf([
+				'danger',
+				'info',
+				'secondary',
+				'success',
+				'warning',
+			]).value('secondary'),
+			selected: Config.bool().value(false),
+			title: Config.string().required(),
+			url: Config.string(),
+		})
+	).required(),
+
+	label: Config.string(),
```

Comment:
Missing JSDOC.

Reply:
In fact I forgot to remove this property, it's inside the items shape, not here.